.github/workflows/tests-typecheck.yml

```
@@ -26,12 +26,8 @@ jobs:
       - name: Install NPM Dependencies
         run: npm install
 
-      - name: Run Type Checking & ESLint
+      - name: Run Prettier, Type Checking, ESLint & Build
```

NIT - I would consider avoiding mentioning the actual technology in general, to make it simpler to change in the future:

```suggestion
      - name: Run formatting, type checking, linting & build
```